packages/account/src/Sections/Verification/ProofOfOwnership/expanded-card.jsx

```diff
@@ -0,0 +1,200 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import FileUploader from './file-uploader.jsx';
+import { Input, Text } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import SampleCreditCardModal from 'Components/sample-credit-card-modal';
+import classNames from 'classnames';
+import { IDENTIFIER_TYPES, VALIDATIONS } from './constants/constants.js';
+
+const ExpandedCard = ({ card_details, error, index, setFieldValue, updateErrors, validateField, values }) => {
+    const [is_sample_modal_open, setIsSampleModalOpen] = useState(false);
+    const handleUploadedFile = (name, file) => {
+        setFieldValue(name, file);
+    };
+    const handleBlur = (name, payment_method_identifier, item_id, item_index) => {
+        handleIdentifierChange(
+            name,
+            formatIdentifier(payment_method_identifier, card_details?.identifier_type),
+            item_id,
+            item_index
+        );
+    };
+    const handleIdentifierChange = (name, payment_method_identifier, item_id, item_index) => {
+        setFieldValue(`${name}`, {
+            ...values.data?.[index]?.[item_index],
+            id: item_id,
+            payment_method_identifier,
+            is_generic_pm: card_details.is_generic_pm,
+            identifier_type: card_details.identifier_type,
+        });
+    };
+    const exampleLink = () =>
+        card_details?.identifier_type === IDENTIFIER_TYPES.card_number && (
+            <span
+                className='proof-of-ownership__card-open-desc-link'
+                key={0}
+                onClick={() => {
+                    setIsSampleModalOpen(true);
+                }}
+            >
+                {localize('See example')}
+            </span>
+        );
+    const formatIdentifier = (payment_method_identifier, identifier_type) => {
+        let formatted_id = payment_method_identifier?.replace(/\s/g, '') ?? '';
```

Comment:
```suggestion
        let formatted_id = payment_method_identifier?.replace(/\s/g, '') || '';
```

if formatted_id is a string, I believe there is no need to have a possibility to accept 0, as this operator allows us to do it